src/dao/TweetDao.ts

```diff
@@ -1,5 +1,5 @@
-import Tweet from "@db/models/Tweet";
```

Comment:
Why this change?

Reply:
Getting error when executing `npm start` from /start in the docker container.

Reply:
filenames you have given in small but here you using title case.

Reply:
```
src/db/models/UserFollower.ts(8,18): error TS2307: Cannot find module '@db/models/User'.
src/db/models/tweet.ts(11,18): error TS2307: Cannot find module '@db/models/User'.
src/db/models/user.ts(14,19): error TS2307: Cannot find module '@db/models/Tweet'.
```

Reply:
Ok For this we should keep CamelCase of ModelNames and similar filenames. This will help us in keeping names as UserFollower.ts

Reply:
Do we need to change the camelCase or TitleCase? Will it be follow for the models files or throughout the app?

Reply:
Correct TitleCase